.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Create Release and Tag
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version for the release"
+        required: true
+        default: "0.1.0"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v3
+
+      - name: Set up Git
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+
+      - name: Create tag
+        id: tag_step
+        run: |
+          git tag -a v${{ github.event.inputs.version }} -m "Release version v${{ github.event.inputs.version }}"
+          git push origin v${{ github.event.inputs.version }}
+
+      - name: Create GitHub Release
+        uses: actions/create-release@v1
+        with:
+          tag_name: v${{ github.event.inputs.version }}
+          release_name: v${{ github.event.inputs.version }}
+          draft: false
+          prerelease: false
+          body: "Release of nova-sdk version v${{ github.event.inputs.version }}"
+
+      - name: Go to module directory
```

Comment:
and then? we should also push the project to pkg.go.dev.
look at [this page](https://pkg.go.dev/about#adding-a-package) to understand how to add our package there

Reply:
No need for that. The Go module is automatically indexed by `pkg.go.dev` once the tagged version is pushed to the repository, and we are already creating a tag in the `.yml` file. You can check this [package](https://pkg.go.dev/github.com/Imad-Allal/pkg-go-dev-demo@v0.1.0) for instance.